### PR TITLE
feat: add articlesConnection to FairOrganizer type

### PR DESCRIPTION
**Pull request by dblandin**
Allow client to query for articles associated with any fairs associated with a fair organizer.

Capped to support at most 100 fairs for a particular fair organizer which should be more than enough to support all current fair organizers.

Jira: [FX-3152]

[FX-3152]: https://artsyproduct.atlassian.net/browse/FX-3152

## Example

**Request**

```graphql
{
  fairOrganizer(id: "the-armory-show") {
    articlesConnection(first: 5, inEditorialFeed: false, sort: PUBLISHED_AT_DESC) {
      totalCount
      edges {
        node {
          title
          publishedAt
          author {
            name
          }
        }
      }
    }    
  }
}
```

**Response**

```json
{
  "data": {
    "fairOrganizer": {
      "articlesConnection": {
        "totalCount": 159,
        "edges": [
          {
            "node": {
              "title": "What Sold during Armory Week",
              "publishedAt": "2020-03-09T17:07:10.641Z",
              "author": {
                "name": "Artsy Editorial"
              }
            }
          },
          {
            "node": {
              "title": "The 10 Best Booths at  The Armory Show",
              "publishedAt": "2020-03-05T18:26:07.868Z",
              "author": {
                "name": "Artsy Editorial"
              }
            }
          },
          {
            "node": {
              "title": "The Armory Show: 2020 Programming",
              "publishedAt": "2020-02-24T18:57:38.785Z",
              "author": {
                "name": "The Armory Show"
              }
            }
          },
          {
            "node": {
              "title": "The Armory Show: 2020 Fair Highlights",
              "publishedAt": "2020-02-24T18:40:53.640Z",
              "author": {
                "name": "The Armory Show"
              }
            }
          },
          {
            "node": {
              "title": "What Sold during Armory Week",
              "publishedAt": "2019-03-11T16:25:06.643Z",
              "author": {
                "name": "Artsy Editorial"
              }
            }
          }
        ]
      }
    }
  }
}
```

### CODE REVIEW

File: src/schema/v2/fair_organizer.ts [hunk 1]
```diff
@@ -23,6 +27,69 @@ export const FairOrganizerType = new GraphQLObjectType<any, ResolverContext>({
...
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const gravityOptions = {
+            fair_organizer_id: profile_id,
+            size: 100,
+          }
+
+          const { body } = await fairsLoader(gravityOptions)
```
Comment (dblandin) [inline src/schema/v2/fair_organizer.ts:59]: Note that when both `articlesConnection` and `fairsConnection` are queried you'll see two requests to the underlying `/api/v1/fairs?fair_organizer_id=[slug]` API endpoint in Gravity.

File: src/schema/v2/fair_organizer.ts [hunk 2]
```diff
@@ -23,6 +27,69 @@ export const FairOrganizerType = new GraphQLObjectType<any, ResolverContext>({
...
+          return {
+            totalCount: count,
+            pageCursors: createPageCursors({ ...args, page, size }, count),
+            ...connectionFromArraySlice(results, args, {
+              arrayLength: count,
+              sliceStart: offset,
+            }),
+          }
+        },
+      },
```
Comment (damassi) [inline src/schema/v2/fair_organizer.ts:92]: @dblandin - This stuff is understandably copy-paste at this point but would still be nice if we could follow-up with some test coverage

Comment (dblandin) [reply src/schema/v2/fair_organizer.ts:92]: >  would still be nice if we could follow-up with some test coverage

Absolutely 👍🏼  I'll get some in tomorrow. Wanted to get these changes out in the limited time I had today to unblock @anastasiapyzhik's work on the new fair organizer page so was a bit aggressive on the commit/push/merge.

Comment (damassi) [reply src/schema/v2/fair_organizer.ts:92]: Np! And thanks for the follow-up 🙏

Comment (dblandin) [reply src/schema/v2/fair_organizer.ts:92]: Follow-up PR here: https://github.com/artsy/metaphysics/pull/3322

**Pull request merged**